### PR TITLE
ci: fix containerfile build for monorepo layout

### PR DESCRIPTION
**Pull request by conorsch**
Follow-up to #167.

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -9,12 +9,13 @@
 
 
 # nextjs files
-!apps/*/src/
-!apps/*/public/
-!apps/*/.next/
-!apps/*/*.ts
+!apps/
```
Comment (conorsch) [inline .dockerignore:12]: The line `!apps/*/src/` looks to me like it should be sufficient to get the `apps/web/src/*` directory allowed, but it wasn't working. To resolve, I added `!apps/` and I'm calling it a day.

File: .github/workflows/container.yml [hunk 2]
```diff
@@ -45,7 +45,7 @@ jobs:
         with:
           context: .
           platforms: linux/amd64
-          file: Containerfile
+          file: apps/web/Containerfile
```
Comment (conorsch) [inline .github/workflows/container.yml:48]: The `Containerfile` moved, but the build "content" did not: the context remains the repo root.

File: .gitignore [hunk 3]
```diff
@@ -9,6 +9,7 @@ node_modules
 .env.development.local
 .env.test.local
 .env.production.local
+.envrc
```
Comment (conorsch) [inline .gitignore:12]: I use a local `.envrc` file sometimes to test connecting a locally-built container to a remote DB URL.

File: apps/web/Containerfile [hunk 4]
```diff
@@ -59,6 +62,6 @@ COPY --from=installer /app/apps/web/package.json .
 # https://nextjs.org/docs/advanced-features/output-file-tracing
 COPY --from=installer --chown=nextjs:nodejs /app/apps/web/.next/standalone ./
 COPY --from=installer --chown=nextjs:nodejs /app/apps/web/.next/static ./apps/web/.next/static
-COPY --from=installer --chown=nextjs:nodejs /app/apps/web/public ./apps/web/public
+# COPY --from=installer --chown=nextjs:nodejs /app/apps/web/public ./apps/web/public
```
Comment (conorsch) [inline apps/web/Containerfile:65]: AFAICT the `./apps/web/public/` dir doesn't exist and doesn't need to exist: the app works well without it.

File: apps/web/Containerfile [hunk 5]
```diff
@@ -4,19 +4,22 @@ LABEL maintainer="team@penumbralabs.xyz"
 
 # provide pnpm globally for dep installing and building
 FROM alpine AS base
+ENV NEXT_TELEMETRY_DISABLED=1
```
Comment (conorsch) [inline apps/web/Containerfile:7]: Porting this no-telem env var across all build stages...

**Review by conorsch — COMMENTED** [review]
Left some descriptive comments in-line, to aid in review.

File: apps/web/Containerfile (hunk 5 above)
Comment (ejmg) [reply apps/web/Containerfile:7]: lol, great catch. Very obnoxious to keep track of.

File: .dockerignore (hunk 1 above)
Comment (ejmg) [reply .dockerignore:12]: I completely assumed that `!apps/*/src` would expand to include any subdirectories so great catch!

**Review by ejmg — APPROVED** [review]

**Comment by ejmg** [comment]
Thanks for the PR and comments 👍

**Pull request merged**